crates/astria-sequencer/src/api_state_ext.rs

```diff
@@ -412,6 +412,8 @@ mod test {
                 amount,
                 asset,
                 destination_chain_address,
+                [0; 32].into(),
```

Comment:
I see why you needed the `From` impl. This entire construction here is iffy and should be reworked, but for the time being you should be able to side-step the issue by just populating a protobuf `raw::Deposit` and converting that.